docker/sandbox-ultra/Makefile

```diff
@@ -0,0 +1,93 @@
+.SILENT: help
+.PHONY: help
+help:
+	echo Available recipes:
+	cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' | awk 'BEGIN { FS = ":.*?## " } { cnt++; a[cnt] = $$1; b[cnt] = $$2; if (length($$1) > max) max = length($$1) } END { for (i = 1; i <= cnt; i++) printf "  $(shell tput setaf 6)%-*s$(shell tput setaf 0) %s\n", max, a[i], b[i] }'
+	tput sgr0
+
+export DOCKER_BUILDKIT := 1
+
+IMAGES := \
+	kubernetesui/dashboard:v2.6.1 \
+	bitnami/minio:2022-debian-11 \
+	postgres:14-alpine \
+	envoyproxy/envoy:v1.23-latest \
+	registry:2
+
+define IMAGE_RULE
+images/tar/$(subst /,-,$(word 1,$(subst :, ,$(1)))):
+	mkdir -p $$(dir $$@)/arm64
+	docker pull --platform linux/arm64 $(1)
+	docker tag $(1) $(word 1,$(subst :, ,$(1))):latest
+	docker save --output $$(dir $$@)arm64/$(subst /,-,$(word 1,$(subst :, ,$(1)))).tar $(word 1,$(subst :, ,$(1))):latest
+
+	mkdir -p $$(dir $$@)/amd64
+	docker pull --platform linux/amd64 $(1)
+	docker tag $(1) $(word 1,$(subst :, ,$(1))):latest
+	docker save --output $$(dir $$@)amd64/$(subst /,-,$(word 1,$(subst :, ,$(1)))).tar $(word 1,$(subst :, ,$(1))):latest
+endef
+
+$(foreach name,$(IMAGES),$(eval $(call IMAGE_RULE,$(name))))
+
+images/tar/flyte:
```

Comment:
We should be building/pushing this separately.

Reply:
meaning ?